### ISSUE TITLE
Thorium vertical Tab bar 

### ISSUE THREAD

**Issue by mohdq123** [Alex313031/thorium#459]
please add a new option for vertical tab bar instead of horizontal tab bar ??

**Comment by Gr-Nobel** [comment]
yes a custom vertical tab bar would be awesome

**Comment by Drakonas** [comment]
I agree. I relied on this so much with my other browsers since I use way too many tabs for horizontal to be viable. Also using it with a vertical monitor view is impossible.

**Comment by zodman** [comment]
same as #419

**Comment by manchbox** [comment]
any news on vertical tabs, I love speed of Thorium after trying Zen, Vivaldi, Brave, but everyone has vertical tabs now - its essential once you've tried them you will never go back

**Comment by desrod** [comment]
Vertical tabs has been a built-in in Chrome for several months, and enabled as an Experimental feature under `chrome://flags` for a  year before that. It seems that Thorium is based on a fork of the Chromium code that predates both of those additions. 

Any reason why vertical tabs can't be built into a Thorium build, given the code already exists in the Chromium source?
